src/main/kotlin/pl/edu/agh/gem/internal/sort/ActivityMerger.kt

```diff
@@ -0,0 +1,52 @@
+package pl.edu.agh.gem.internal.sort
+
+import pl.edu.agh.gem.internal.model.finance.Activity
+import pl.edu.agh.gem.internal.model.finance.filter.FilterOptions
+import pl.edu.agh.gem.internal.model.finance.filter.SortOrder.ASCENDING
+import pl.edu.agh.gem.internal.model.finance.filter.SortOrder.DESCENDING
+import pl.edu.agh.gem.internal.model.finance.filter.SortedBy.DATE
+import pl.edu.agh.gem.internal.model.finance.filter.SortedBy.TITLE
+
+class ActivityMerger(
```

Comment:
we dont want to have that as a @service or other component to not create that each time ?